quodlibet/po/fr.po

```diff
@@ -4409,10 +4408,12 @@ msgid ""
 "The selected songs will be removed from the library and their files deleted "
 "from disk."
 msgstr ""
+"Les fichiers sélectionnés seront supprimés de la bibliothèque et leurs "
```

Comment:
The first occurrence  of "fichiers" could be replaced by "titres", to avoid using the same word for representing two different things.